wpilibc/src/main/native/include/ErrorBase.h

```diff
@@ -7,6 +7,8 @@
 
 #pragma once
 
+#include <set>
+
```

Comment:
It would happen whenever the user does something that requires calling wpi_setWPIError() or others. That's hopefully rare, if at all.